images/capi/ansible/roles/sysprep/tasks/redhat.yml

```diff
@@ -19,6 +19,25 @@
 
 - import_tasks: rpm_repos.yml
 
+- name: set EPEL repo location
+  set_fact:
+    epel_repo_file: /etc/yum.repos.d/epel.repo
+  when: not disable_public_repos|bool
+
+- name: set EPEL repo location
+  set_fact:
+    epel_repo_file: /etc/yum.repos.d/epel.repo.disabled
+  when: disable_public_repos|bool
+
+# EPEL should not be left enabled for RHEL
+- name: disable EPEL repo on RHEL
```

Comment:
I'm going to have to think on this one just a bit more. I understand the point -- leaving EPEL enabled for python2-pip, which is now in the provider role instead of `node`, but we really only want to leave it enabled for that one single package/step.